examples/src/features/authorization.ts

```diff
@@ -5,7 +5,6 @@ const isBrowser = false;
 
 (async () => {
```

Comment:
```suggestion
await (async () => {
```
Missing await at the top level of the code to make it synchronous